.github/workflows/ci.yml

```diff
@@ -215,7 +217,7 @@ jobs:
 
       - name: Download LLVM Binaries
         id: llvm-download
-        if: steps.llvm-cache.outputs.cache-hit != 'true'
+        if: steps.llvm-cache.outputs.cache-hit != 'true' && !(matrix.ubsan || matrix.asan || matrix.msan)
```

Comment:
Ideally, we should also upload these binaries. The binaries on the website are not only useful for developers. They're also helpful when testing CI, and the GHA cache is not yet available.